docs/design/design-patterns/non-functional-requirements-capture-guide.md

```diff
@@ -0,0 +1,149 @@
+# Non-Functional Requirements Capture
+
+## Goals
+
+In software engineering projects, important characteristics of the system providing for necessary e.g., testability, reliability, scalability, observability, securability, manageability are best considered as first-class citizens in the requirements gathering process.
+By defining these non-functional requirements in detail early in the engagement, they can be properly evaluated when the cost of their impact on subsequent design decisions is comparatively low.
+
+To support the process of capturing a project's _comprehensive_ non-functional requirements, this document offers a taxonomy for non-functional requirements and provides a framework for their identification, exploration, assignment of customer stakeholders, and eventual codification into formal engineering requirements as input to subsequent solution design.
+
+---
+
+## Areas of Investigation
+
+- [Enterprise Security](../../security/README.md)
```

Comment:
Nit: I can see a gap between Enterprise Security and the next indented section of content, is there any way to remove this? Suggest looking at how we can make this as clear as possible formatting wise.

Reply:
If I'm understanding the comment correctly, you are calling attn to a repeating rendering oddity that seems to manifest in _all_ elements at that same top-most outline level e.g., `Enterprise Security`, `Enterprise AuthN/AuthZ`, `Enterprise Monitoring/Operations`, `Other standard Enterprise technologies/practices`, `Production Ecosystem`.

If there is a 'trick' to control this in markdown, happy to introduce it, but as far as I can tell, this is an in-built behavior of markdown renderers (?)

Alternately, perhaps a more visually appealing strategy might be to have those top-most elements in the outline actually be separate third-level headings instead of the whole thing being just a single perpetual outline structure.  Something along the lines of...

## Areas of Investigation

### [Enterprise Security](../../security/README.md)
  - Privacy
    - PII
    - HIPAA
  - Encryption
  - Data mobility
    - at rest
    - in motion
    - in process/memory
...
### Enterprise AuthN/AuthZ
  - Users
  - Services
  - Authorities/directories
...
### [Enterprise Monitoring/Operations](../../observability/README.md)
  - Logging
    - Operations
    - Reporting
    - Audit
  - Monitoring
    - Diagnostics/Alerts
...
### Other standard Enterprise technologies/practices
  - Developer ecosystem
    - Platform/OS
      - Hardened
      - Approved base images
...
### Production ecosystem
  - Platform/OS
    - Hardened
    - Approved base images
...

In the above, you will note that the markdown renderer still places increased whitespace after each "###" heading, but its more visually palettable because they are explicitly _headings_ instead of just "level 1 bullets".

Also, separating what is now one singular over-long outline into what is effectively multiple smaller 'topic/SME-area-focused' outlines might make the content easier for a reader to consume (?)

Thoughts --?

Reply:
@sbohlen I really like this suggestion, pulling the top-most elements as headers definitely improves the formatting 💯

Reply:
Done.